modules/oadp-features.adoc

```diff
@@ -9,13 +9,26 @@
 OpenShift API for Data Protection (OADP) supports the following features:
 
 Backup::
-You can back up all resources in your cluster or you can filter the resources by type, namespace, or label.
+You can use OADP to back up all customer applications on the OpenShift Platform, or you can filter the resources by type, namespace, or label.
 +
 OADP backs up Kubernetes objects and internal images by saving them as an archive file on object storage. OADP backs up persistent volumes (PVs) by creating snapshots with the native cloud snapshot API or with the Container Storage Interface (CSI). For cloud providers that do not support snapshots, OADP backs up resources and PV data with Restic.
 
++
+[NOTE]
+====
+OADP may not successfully restore Operators, if it is not appropriately configured. Operators generated resources with OwnerReferences to another Operator custom resource (CR) should generally be excluded, and allow the Operator to recreate them for restore to be successful.
+====
+
```

Comment:
The IBN Style guide recommends "might" instead of "may" to remove ambiguity. "It" is ambiguous, so i suggest replacing it with OADP. The second sentence is not clear. Are you saying that it is recommended to avoid using Operator-generated resources that have `OwnerReferences` that point to another Operator custom resource (CR)"? I don't understand the last clause. Avoid "should" -- either use "must/have to" or say 'Do this". Take a look at "recommend" in https://redhat-documentation.github.io/supplementary-style-guide/#project for more ideas.